packages/harmony/src/components/input/SelectablePill/SelectablePill.tsx

```diff
@@ -0,0 +1,46 @@
+import { forwardRef } from 'react'
+
+import cn from 'classnames'
+
+import { Text } from 'components/typography'
+
+import styles from './SelectablePill.module.css'
+import type { SelectablePillProps } from './types'
+
+export const SelectablePill = forwardRef<
+  HTMLButtonElement,
+  SelectablePillProps
+>(
+  (
```

Comment:
can we do `(props, ref) =>` to make this a bit easier to read?

Reply:
it was originally like that, but story book does not detect default props if we destructure the props in the component. kind of lame, but ill see if i can get it to work

Reply:
Oh super interesting, good to know